### DISCUSSION TITLE
Update World Events that rotate around the calendar to better match future years

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#6834] [Suggestions]
On CC, this year we had Lunar Festival start on 1/23/24 and end on 2/5/24, yet the Lunar New Year was actually on 2/10/24 IRL this year.  My assumption is that the code is still pulling from what were correct dates back when Wrath was active on Blizzard servers, and haven't been meaningfully updated since then.

The majority of world events are based on static dates (Hallow's End, Feast of Winter Veil, etc.) so they don't fall out of sync, but there are a handful that are based on lunar cycles and religious calendars.  For future years, could we update these rotating holidays to better match IRL and/or Classic/Retail Blizzard?  

Lunar Festival
Noblegarden
Harvest Festival (potentially)

**Comment by 55Honey** [comment]
While you're probably right that some festivities where related to the lunar calendar, it is not feasible to change that calculation for the simple reason, that sooner or later these festivities would overlap with other events.

**Reply by heyitsbench** [reply]
> [...] sooner or later these festivities would overlap with other events.

So what? They do on Blizz servers, why shouldn't they on AC/CC?

**Reply by 55Honey** [reply]
If that is true, we don't need a discussion, but a bug-report about it. 
Then it is to be decided if we want the original dates from the 3.3.5 era or an annual adjustment based on the lunar calendar.

**Reply by andovcr** [reply]
CC already has overlapping events - obviously the fishing extravaganzas and the Darkmoon Faire on regular cycles, but even holidays like Harvest Festival & Brewfest.

I opted to make this a suggestion, because if this is a "decision" to use the original dates or adjustments, then that's not a bug - that's not something that can be triaged.  Instead, this would be a quality of life improvement to make the world event dates more logical in future years.  Isn't that the very definition of a discussion?  If not, could you please elaborate on why this would be better submitted as a bug?

<img width="495" alt="image" src="https://github.com/chromiecraft/chromiecraft/assets/58115742/501229d4-0004-455f-9e85-3ade2a2a201b">

**Comment by Amrx2** [comment]
Whilst a valid suggestion, due to the leap-year it would appear events are out of allignment with how they should be (noblegarden has commenced already in classic but is another few weeks away on Chromiecraft) this is a bug and needs to be raised in that manner. Once raised can get this looked at.
